packages/api/repository/EnvironmentRepository.ts

```diff
@@ -49,8 +50,20 @@ export class EnvironmentRepository extends Repository<Environment> {
   }
 
   // PUT update environment
-  updateEnvironment(id: string, changes: IEnvironment): Promise<UpdateResult> {
-    return this.update(id, { ...changes });
+  async updateEnvironment(id: string, overwrites: IEnvironment): Promise<Environment> {
+    await this.update(id, {
+      administrators: [],
+      contributors: [],
+      readOnlyOperators: [],
+      ...overwrites,
```

Comment:
Does this reset the adminstrators to []?

Reply:
Well it depends on what is sent with the request body. If no `administrators` are sent then yes the administrators are reset (same with all the operators). I went with replacing all operators because I was unsure if the operators should appended to the previous values or be replaced entirely. 

I decided on replacing because if some operators are removed (not included in the request) and others added, I assumed we did not want to keep the older operators. I think I see where you are coming from and we should iron out it should be handled this way or not.